e2e/test/scenarios/embedding/embedding-native.cy.spec.js

```diff
@@ -337,15 +338,22 @@ describe("scenarios > embedding > native questions with default parameters", ()
       });
     });
 
-    visitIframe();
-
-    // Remove default filter value
-    clearFilterWidget();
-    // The ID default (1, 2) should apply, because it is disabled.
-    // The Name default ('Lina Heaney') should not apply, because the Name param is editable and empty
-    // The Source default ('Facebook') should not apply because the param is locked but the value is unset
-    // If either the Name or Source default applied the result would be 0.
-    cy.findByTestId("scalar-value").invoke("text").should("eq", "2");
+    cy.get("@questionId").then(questionId => {
+      const payload = {
+        resource: { question: questionId },
+        params: { source: [] },
+      };
+
+      visitEmbeddedPage(payload);
+
+      // Remove default filter value
+      clearFilterWidget();
+      // The ID default (1, 2) should apply, because it is disabled.
+      // The Name default ('Lina Heaney') should not apply, because the Name param is editable and empty
+      // The Source default ('Facebook') should not apply because the param is locked but the value is unset
+      // If either the Name or Source default applied the result would be 0.
+      cy.findByTestId("scalar-value").invoke("text").should("eq", "2");
```

Comment:
Can you add a test that asserts a similar error message in the dashboard `There was a problem displaying this card`, but for question?

![image](https://github.com/user-attachments/assets/3965527a-4a22-406c-a31b-839c0ee1acfb)